src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton4() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton4.isDisplayed());
+ 	 		clickUpdateButton4.click();
+ 	 		JavaScriptExec.sleep();
+ 	 		}
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--12-update-form']/div[2]/a[1]")
+ 	 //*[@id="change--12-update-form"]/div[2]/a[1]/span
+ 	 	private WebElement clickUpdateButton5;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton5() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton5.isDisplayed());
+ 	 		clickUpdateButton5.click();
+ 	 		JavaScriptExec.sleep();
+ 	 		}
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6-update-form']/div[2]/a[1]")
+ 	 	//*[@id="change--6-update-form"]/div[2]/a[1]/span
+ 	 	private WebElement clickUpdateButton6;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton6() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton1.isDisplayed());
+ 	 		clickUpdateButton1.click();
+ 	 		JavaScriptExec.sleep();
+ 	 		}
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6-update-form']/div[2]/a[1]")
+ 	 	//*[@id="change--6-update-form"]/div[2]/a[1]/span
+ 	 	private WebElement clickUpdateButton7;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton7() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton1.isDisplayed());
+ 	 		clickUpdateButton1.click();
+ 	 		JavaScriptExec.sleep();
+ 	 		}
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--5-update-form']/div[2]/a[1]")
+ 	 //*[@id="change--5-update-form"]/div[2]/a[1]/span
+ 	 	private WebElement clickUpdateButton8;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton8() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton8.isDisplayed());
+ 	 		clickUpdateButton8.click();
+ 	 		JavaScriptExec.sleep();
+ 	 		}
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--5-update-form']/div[2]/a[1]")
+ 	
+ 	 //*[@id="change--5-update-form"]/div[2]/a[1]/span
+ 	 	private WebElement clickUpdateButton12;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
```

Comment:
@mayankchhajer Please remove the xpaths after pasting and format the code

Reply:
Done